src/styles/global.css

```
@@ -14,326 +14,339 @@
+    --colour-link-text: #0284c7;
 }
 :root:not(.dark)[data-accent="neutral"] {
-  --colour-accent-primary: #52525b;
-  --colour-accent-hover: #3f3f46;
-  --colour-link-text: #52525b;
+    --colour-accent-primary: #52525b;
+    --colour-accent-hover: #3f3f46;
+    --colour-link-text: #52525b;
 }
 /* Fallback for pages without data-accent (backwards compat) */
 :root:not(.dark):not([data-accent]) {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)